Plugins/Power Manager/Devices & Monitoring List/PowerShell/Find_non_PMP_capable_devices.ps1

```diff
@@ -0,0 +1,137 @@
+<#
```

Comment:
May be easier to combine this script and the next into one single script - <script name> -ip <x> -credentials <y> -not_capable_of_x -not_capable_of_y ....

Also Powershell scripts (and cmdlets to be when exposed as modules) need to be named in Verb-Noun form ...